README.md

```diff
@@ -91,14 +92,22 @@ Follow the steps [here](https://developer.github.com/apps/building-github-apps/c
 
 ## Running
 
-### Geth
-Launch a local geth node with the bot account unlocked:
+### Ethereum node
+There are two options for connecting to an Ethereum node: either run a local node or connect to a remote one like Infura.
```

Comment:
Geth can also be in remote, so I would probably say `either Geth or Infura`.

Reply:
Changed.